### DISCUSSION TITLE
Работа поля типа даты в быстрой форме

### DISCUSSION THREAD

**Discussion by nshy** [totumonline/totum-ru-issues-and-discussions#502] [ОШБК!] [answered]
Привет! 

Есть проблема с полем такой формы. Все работает, если даты выбирать через календарик. Но многие при заполнении формы вводят дату с клавиатуры. Ее можно ввести, но при подаче формы появляется ошибка, что это поле не заполнено. Наблюдал такую проблему на версии `3.7.*` При обновлении до последней версии (`4.11.57.0`) проблема осталась.

Также есть меньшая проблема, которую тут тоже можно упомянуть. При редактировании даты с клавиатуры при нажатии `backspace` курсор сначала перемещается в конец поля даты и только после повторного нажатия перемещается на нужное место (перед символом, который был удален).

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Надо исправлять, когда это будет не знаю. Точно уже только в PRO в 5
